app/src/main/java/com/example/marvelapp/data/service/response/CharacterEntity.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.marvelapp.data.service.response
+
+open class CharacterEntity(
```

Comment:
where do you use this? if it is a service response should have the **SerializedName** if not should be in domain

Reply:
Thx, you were right, i already have one in the domain folder, and i did not use this. Now its gone.